src/jquery.sceditor.js

```diff
@@ -69,8 +69,8 @@ define(function (require) {
 
 		options = options || {};
 
-		if (!options.runWithoutWysiwygSupport && !browser.isWysiwygSupported) {
-			return;
+		if (options.runWithoutWysiwygSupport || !browser.isWysiwygSupported) {
+			SCEditor.isWysiwygSupported = false;
```

Comment:
I misinterpreted it, then...
Isn't that an option to force it to run without wysiwyg support?